sim/components/ble/SimpleWeatherService.h

```diff
@@ -80,11 +80,18 @@ class SimpleWeatherService {
       int16_t minTemperature;
       int16_t maxTemperature;
       Icons iconId;
+
+      bool operator==(const Day& other) const;
```

Comment:
where does this function come from? I don't think it is needed for the current implementation, and it is not in the current InfiniTime version of `SimpleWeatherService.h`